botfront/private/defaults/private.dev.yaml

```diff
@@ -21,16 +21,7 @@ policies:
     max_history: 5
     batch_size: [32, 64]
   - name: RulePolicy
-    enable_fallback_prediction: false
   - name: AugmentedMemoizationPolicy
-  - name: rasa_addons.core.policies.BotfrontDisambiguationPolicy
-    fallback_trigger: 0.30
-    disambiguation_trigger: '$0 < 2 * $1'
-    disambiguation_template: 'utter_disambiguation'
-    n_suggestions: 2
-    excluded_intents:
-      - ^chitchat\..*
-      - ^basics\..*
```

Comment:
If @GuillaumeTech confirms `BotfrontDisambiguationPolicy`  works with latest Rasa, we should leave it

Reply:
Yes the bf disambiguation work fine with the lastest rasa https://www.notion.so/godialogue/8c37f360c4f148fdb89abc7cb106a4d8?v=466d1829d04a4b3eb2e5e3f2d9dfda70&p=9ac119c2e23847ff82dda5344f915008

Reply:
You haven't tried with both stories and rules.

Reply:
I had issues when I tried.

Reply:
ohh sorry my bad I thought the problem was showing when there was only stories, I'll try with both

Reply:
@pheel @znat No issue when mixing rules and stories !
https://user-images.githubusercontent.com/27060431/107054954-bc810d80-679e-11eb-8f19-e6931e07717a.mov